examples/cryptocurrency/src/lib.rs

```diff
@@ -39,410 +37,451 @@ extern crate router;
 extern crate bodyparser;
 extern crate iron;
 
-// Import necessary types from crates.
-
-use exonum::blockchain::{Blockchain, Service, Transaction, ApiContext, ExecutionResult,
-                         TransactionSet, ExecutionError};
-use exonum::encoding::serialize::FromHex;
-use exonum::node::{TransactionSend, ApiSender};
-use exonum::messages::{RawTransaction, Message};
-use exonum::storage::{Fork, MapIndex, Snapshot};
-use exonum::crypto::{Hash, PublicKey};
-use exonum::encoding;
-use exonum::api::{Api, ApiError};
-use iron::prelude::*;
-use iron::Handler;
-use iron::status::Status;
-use iron::headers::ContentType;
-use iron::modifiers::Header;
-use router::Router;
-
-// // // // // // // // // // CONSTANTS // // // // // // // // // //
+/// Constants
+pub mod constants {
```

Comment:
I think that it would be great to get rid of `constants` module.
`SERVICE_ID` constant can be moved to `service` module and `INIT_BALANCE` - to `contracts` mod.

Reply:
Makes a lot of sense. This is similar to what we see in `exonum-configuration` service. Will change.